aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -330,6 +332,11 @@ CREATE_BINARY_META_FUNC(igamma);
 CREATE_BINARY_META_FUNC(igammac);
 CREATE_BINARY_META_FUNC(nextafter);
 
+// TODO: should these be ternary, not binary?
+// signature: const Tensor& grad_output, const Tensor& self, const Tensor& other
+CREATE_BINARY_META_FUNC(igamma_self_backward);
```

Comment:
I take it the ternary version would be equal to 
```
grad_output * calc_igamma_grada(self, other)
```

It would likely perform better as a fused kernel, but I don't think it's too important either way. Up to you.

Reply:
Yes, equivalent to what you have above, then replace in `derivatives.yaml`:
```
self: grad * igamma_self_backward(self, other)
```
--> 
```
self: igamma_self_backward(grad, self, other)
```
But, this requires adding a `build_borrowing_ternary_op` to `TensorIterator.cpp`, and then, pedantically, probably moving these function definitions from `BinaryOps.cpp` to something else (like `TernaryOps.cpp`, which isn't a thing yet). Seems like an awful lot of trouble/change, unless of course the performance really does matter. The change isn't difficult, just seems like too much.

Reply:
You can use `TensorIteratorConfig` to build the iterator yourself, it doesn't need to be built by `TensorIterator`.

Again, I don't think this is a performance-critical operation though so I'm fine either way.

Reply:
Can you delete the comment?